docs/sources/panels/visualizations/bar-chart.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "Bar chart"
+description = "Bar chart visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "barchart"]
++++
+
+# Bar chart visualization
+
+This panel is focused on visualizing categorical data.
+
+{{< figure src="/static/img/docs/v8/bar_chart_example.png" max-width="1025px" caption="Bar chart example" >}}
+
+## Supported data formats
+
+Only one data frame is supported and it needs to have at least one string field that will be used as the category for an X or Y axis and one or more numerical fields.
+
+Example:
+
+| Browser | Market share |
+| ------- | ------------ |
+| Chrome  | 50           |
+| IE      | 17.5         |
+
+If you have more than one numerical field the panel will show grouped bars.
+
+### Visualizing time series or multiple result sets
+
+If you have multiple time series or tables you first need to join them using a join or reduce transform. For example if you 
+have multiple time series and you want to compare their last and max value add the **Reduce** transform and specify  **Max** and **Last** as options under **Calculations**. 
+
+{{< figure src="/static/img/docs/v8/bar_chart_time_series.png" max-width="1025px" caption="Bar chart time series example" >}}
```

Comment:
Please put images in a subject folder, such as bar-chart-panel, rather than a version folder. Also, we need the Grafana version in the image name, per the [Grafana file naming conventions](https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-style-guide.md#file-naming-conventions).

Reply:
done